tests/steps/other_v2_steps.py

```diff
@@ -97,6 +98,31 @@ def read_program(context, path):
     return read_program_binary(path)
 
 
+# Send transactions to progress block numbers on dev mode.
+def send_zero_transactions(context, txns=1):
```

Comment:
nit: I was a bit confused by the name of this function, thinking maybe it meant "send zero algos" or "send no transactions". So perhaps consider a more descriptive name such as "burn_algo_transactions(context, num_txns=1)". (also reccomend changing the parameter name)

Reply:
Good point - done!